.circleci/config.yml

```diff
@@ -22,11 +22,11 @@ executors:
   test-executor:
     working_directory: /go/src/github.com/klaytn/klaytn
     docker:
-      - image: klaytn/build_base:1.2-go.1.20.6-solc0.8.13
+      - image: klaytn/build_base:1.2-ubuntu18.04-go.1.20.6-solc0.8.13
```

Comment:
@Sotatek-TinnNguyen We don't have to use ubuntu18.04 for test-executor, isn't it? I think it is better to revert rather than fixing ubuntu18.04 image to make solidity working well.

Reply:
hmm, it can be but we will not have consistency between test phase and release phase. I will change as you said but I think we should resolve it later

Reply:
Sure, is it possible to add TODO in this file? If possible, please add a TODO.

Reply:
@Sotatek-TinnNguyen  Sorry to change the policy, but I have discussed with others and concluded that we have to resolve it as you said. Could you investigate what the problem is?

Reply:
Okay, I will check it with you